dags/utils/slack_operator.py

```diff
@@ -1,88 +1,171 @@
-from airflow.hooks.base_hook import BaseHook
-from airflow.contrib.operators.slack_webhook_operator import SlackWebhookOperator
+import os
+import datetime
+from pendulum import timezone
+
+from cron_descriptor import get_description
+from airflow.hooks.base import BaseHook
+from airflow.providers.slack.operators.slack_webhook import SlackWebhookOperator
 
 # This is the Conn Id that we set when creating the connection in the Airflow dashboard
 # in Admin > Connections.
 SLACK_CONN_ID = "slack"
 
+DEPLOYMENT_ENVIRONMENT = os.getenv("ENVIRONMENT", "development")
+
+
+def format_schedule(schedule_interval):
+    """
+    Convert a schedule interval to a human-readable format.
+
+    This function handles different types of schedule intervals:
+    - None: Returns "None" as the description.
+    - str: Assumes the string is a cron expression and tries to convert it to a human-readable format.
+           If conversion fails, it returns the cron expression as is.
+    - datetime.timedelta: Converts the timedelta to a human-readable string, breaking it down into days, hours, minutes, and seconds.
```

Comment:
awesome docstring 🙏 would you mind breaking these lines to keep them under 90ish chars?

Reply:
For sure; I'll go 80 characters for the classic terminal fit. https://github.com/cityofaustin/atd-airflow/pull/261/commits/2f7c9bdd76bb26a93b93f8544a7eb8fec0f2b8e8